src/igvfd/types/ontology_term.py

```diff
@@ -82,6 +82,17 @@ def synonyms(self, registry, term_id):
     def ancestors(self, registry, term_id):
         return self._get_ontology_slims(registry, term_id, 'ancestors')
 
+    @calculated_property(schema={
+        'title': 'Ontology',
+        'type': 'string',
+        'notSubmittable': True,
+    })
+    def ontology(self, properties=None):
+        if properties is None:
+            properties = self.upgrade_properties()
+        format_ontology = properties['term_id'].split(':')
```

Comment:
why not use one liner here? 
`return properties['term_id'].split(':')[0]`

Reply:
Done